### DISCUSSION TITLE
Is this package still maintained?

### DISCUSSION THREAD

**Discussion by rap2hpoutre** [plouc/nivo#1641] [General]
Thank you for this awesome package! I really enjoy it, it also helped me creating products 🙏 Is it still maintained?

**Comment by jamesdh** [comment]
Lots of people starting to wonder this. I'd really prefer to avoid making a fork and restarting this as a new project, but it's starting to look like that might be the only option. 

@plouc or @wyze would you guys at all be up for adding more maintainers? I would be willing to help out if so.

**Comment by rap2hpoutre** [comment]
Adding maintainers could be a good option @plouc and @wyze: some problems are clearly identified (such as `recompose` package usage) and some others already have solutions with pending PR: https://github.com/plouc/nivo/pull/1480. A reboot could be an option too (fork is not as visible as a reboot IMO), if @plouc and @wyze are not available; I would really like to know their opinion. This package is awesome 🎉  I would love to see it continue its life!

**Comment by wyze** [comment]
Sorry, I haven't had enough time to devote to open source lately. I'll try my best to go through open PRs and review them and get a release out. Unfortunately, I don't have access to add other maintainers.

**Comment by plouc** [comment]
Same as @wyze, I've been super busy with work and couldn't spend any time on the project, and I can definitely understand the concern.

I don't plan to let the project die, but on the other hand, I cannot predict how much time I'll be able to invest in the future, I usually work on it in batch (typically holidays), it's not ideal for maintenance and can be quite frustrating for contributors.

I'm open to the idea of adding new maintainers, it would just require some synchronization, we had several informal discussion with @wyze when he joined the project, I guess writing some kind of guidelines, long-term vision could be useful.

I might be able to spend a few days in 2 weeks, so I'll try to tackle the most critical PRs at least and rollout a new release, something I planned to do earlier, but no time unfortunately...

So to answer the question, it's still somehow maintained but time is definitely an issue :/

**Comment by jamesdh** [comment]
@rap2hpoutre agreed, this package is way too awesome to let it go stale! 

I'm not a d3 expert by any means, but I'm capable enough that I'd feel comfortable with reviewing critical PR's, help getting them merged, and releases out the door, if need be.
